onnxruntime/test/onnx/TestCase.cc

```diff
@@ -210,6 +210,8 @@ class OnnxModelInfo : public TestModelInfo {
     const std::regex onnx_tag_regex("onnx[0-9a-z]{3}"); //e.g. onnx141, onnx150, onnxtip
     if (std::regex_search(url_string, match, onnx_tag_regex)) {
       onnx_commit_tag_ = match[0].str();   
+    } else {
+      onnx_commit_tag_ = "unknown version";
```

Comment:
>"unknown version" [](start = 25, length = 17)

Please, define a std::string constant and refer to it in both places. This way you do not depend on the exact spelling of the string.

Reply:
Good idea, let me do it.